src/content/contribute/writing-a-loader.md

```diff
@@ -333,15 +333,15 @@ export default (fixture, options = {}) => {
         test: /\.txt$/,
         use: {
           loader: path.resolve(__dirname, '../src/loader.js'),
-          options: {
-            name: 'Alice'
-          }
+          options,
         }
       }]
     }
   });
 
-  compiler.outputFileSystem = createFsFromVolume(new Volume());
+  const volume = createFsFromVolume(new Volume());
+  volume.join = path.join;
```

Comment:
https://github.com/streamich/memfs/issues/404
https://github.com/webpack/memory-fs/issues/67

Reply:
Without this, it throws error: `TypeError: this.outputFileSystem.join is not a function`